sklearn/ensemble/_hist_gradient_boosting/splitting.pyx

```diff
@@ -1,11 +1,13 @@
 """This module contains routines and data structures to:
 
+# Authors: The scikit-learn developers
```

Comment:
Looking at other places, it seems the authors+licence should not be inside the docstring but after the docstring?

Reply:
Oh yes, that's true.